services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/geometria/GeometriatServiceImplITest.kt

```diff
@@ -41,10 +48,8 @@ internal class GeometriatServiceImplITest : DatabaseTest() {
 
     @Test
     fun `save and load and update`() {
-        val geometriat =
-            "/fi/hel/haitaton/hanke/geometria/hankeGeometriat.json".asJsonResource(
-                Geometriat::class.java
-            )
+        val geometriat: Geometriat =
+            "/fi/hel/haitaton/hanke/geometria/hankeGeometriat.json".asJsonResource()
```

Comment:
Could these be a separate function somewhere? Either a private function for this class or somewhere else where it could be accessed by other test classes.

Reply:
I added a factory class for creating the geometries.